oauth2client/contrib/metadata.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Well, I can imagine people wanting a thin wrapper like this for non OAuth related reasons but I guess that doesn't really belong in the package. Can more to private